apps/api/package.json

```diff
@@ -36,7 +39,7 @@
     "comp:status": "tsx scripts/competition-status.ts",
     "generate-openapi": "tsx -e \"const { swaggerSpec } = require('./src/config/swagger'); require('fs').writeFileSync('./openapi/openapi.json', JSON.stringify(swaggerSpec, null, 2));\"",
     "generate-markdown": "openapi-markdown -i ./openapi/openapi.json -o ./openapi/API.md",
-    "generate-docs": "npm run generate-openapi && npm run generate-markdown"
+    "generate-docs": "pnpm generate-openapi && pnpm generate-markdown"
```

Comment:
I noticed that this script hasn't been run, but the spec was changed.

Reply:
Not sure. I never ran this script and I never updated the spec in any way. Let me see what the spec change was.